cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Hash.hs

```diff
@@ -0,0 +1,50 @@
+
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE GADTs #-}
+
+module Cardano.CLI.EraBased.Options.Governance.Hash
+  (
+    pGovernanceHashCmds
+  ) where
+
+import           Cardano.Api
+
+import           Cardano.CLI.EraBased.Commands.Governance.Hash
+                   (GovernanceHashCmdArgs (GovernanceHashCmdArgs))
+import qualified Cardano.CLI.EraBased.Commands.Governance.Hash as Cmd
+import           Cardano.CLI.EraBased.Options.Common
+
+import           Data.Foldable
+import           Options.Applicative
+import qualified Options.Applicative as Opt
+
+pGovernanceHashCmds :: ()
+  => CardanoEra era
+  -> Maybe (Parser (Cmd.GovernanceHashCmds era))
+pGovernanceHashCmds era = do
+  eon <- forEraMaybeEon era
+  return
+    $ subParser "hash"
+    $ Opt.info
+        ( Cmd.GovernanceHashCmd . GovernanceHashCmdArgs eon
+            <$> pGovernanceHashSource
+        )
+    $ Opt.progDesc "Hash governance data."
```

Comment:
I had written something minimalist, because the other commands were like this, but you have a good point :+1: 

So I changed to _Compute the hash to pass to the various --*-hash arguments of governance commands._ (to account that this command will be used also for a non `--anchor-data-hash` argument, once https://github.com/input-output-hk/cardano-cli/issues/447 is tackled).